lib/modules/manager/pipenv/readme.md

```diff
@@ -1 +1,6 @@
 `Pipenv.lock` updating is supported.
+
+The following `depTypes` are currently supported by the pipenv manager:
```

Comment:
```suggestion
The following `depTypes` are supported by the Pipenv manager:
```

Let's leave out `currently`, our readers will expect us to document things that are currently supported by default. 😉 
Also capitalize the proper noun Pipevn.